### PR TITLE
Add Heart Languages

### PR DESCRIPTION
**Pull request by AnonymousWalker**
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/85)

### CODE REVIEW

**Comment by AnonymousWalker** [comment]
___
*[fetcher-web/src/main/kotlin/org/bibletranslationtools/fetcher/impl/repository/LanguageRepositoryImpl.kt, line 9 at r8](https://reviewable.io/reviews/bible-translation-tools/fetcher/85#-MNP7Dqc8-1BXDvod7Ll:-MNP7Dqc8-1BXDvod7Lm:b-b5ybm5) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/be07e9ad9691c1db38220f9d69f959e4f62598e1/fetcher-web/src/main/kotlin/org/bibletranslationtools/fetcher/impl/repository/LanguageRepositoryImpl.kt#L9)):*
> ```Kotlin
> class LanguageRepositoryImpl(
>     private val gatewayCatalog: LanguageCatalog,
>     private val heartCatalog: LanguageCatalog
> ```

This is welcome for any suggestion, since we are expecting to have a single Language Repository that can serve out different language groups or all of them (?)

**Review by jsarabia — COMMENTED** [review]
> Reviewed 1 of 8 files at r1, 2 of 15 files at r4, 2 of 13 files at r5, 6 of 21 files at r8.
> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/85)* status: 10 of 31 files reviewed, all discussions resolved (waiting on @16dprice and @jsarabia)

___
*[fetcher-web/src/main/kotlin/org/bibletranslationtools/fetcher/impl/repository/LanguageRepositoryImpl.kt, line 9 at r8](https://reviewable.io/reviews/bible-translation-tools/fetcher/85#-MNP7Dqc8-1BXDvod7Ll:-MNPOsfj5L1QglowFJ-e:b-78e0kl) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/be07e9ad9691c1db38220f9d69f959e4f62598e1/fetcher-web/src/main/kotlin/org/bibletranslationtools/fetcher/impl/repository/LanguageRepositoryImpl.kt#L9)):*
<details><summary><i>Previously, AnonymousWalker (Tony) wrote…</i></summary><blockquote>

This is welcome for any suggestion, since we are expecting to have a single Language Repository that can serve out different language groups or all of them (?)
</blockquote></details>

We may choose to keep it this way for now, since unfoldingWord's GL list might differ from ours, but AFAIK we use their HL list. Two different LanguageCatalogs allows for two separate implementations pretty easily.

Not that we couldn't have a languageCatalog class that is some combination of the two; or some kind of aggregate class which itself implements the Catalog interface but delegates to the two different ones, but yeah. 

Basically saying this is fine.

**Review by jsarabia — DISMISSED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/85#-:-MNPngKu7skN2Tuf9t0x:bnfp4nl)* status: 10 of 31 files reviewed, all discussions resolved (waiting on @16dprice and @jsarabia)
